cobbler/package/clean.sls

```diff
@@ -9,6 +9,12 @@
 include:
   - {{ sls_config_clean }}
 
+{% if pkg.communityrepo.enabled %}
+cobbler-package-clean-repository-removed:
+  pkgrepo.absent:
+    - name: cobbler_community_repo
+{% endif %}
```

Comment:
I know we talked about this in the other PR - https://github.com/cobbler/cobbler-formula/pull/4#discussion_r1372305802

Please resolve the discussion here if you have no further questions. If you have let's discuss it before we merge this PR.

Reply:
I got the reason for this now, still figure it would make sense to clean _all_ the Cobbler remains regardless of the currently enabled pillar settings? But it's a detail for later. :)